buildSrc/src/main/kotlin/java-library-conventions.gradle.kts

```diff
@@ -50,6 +50,7 @@ if (project in mavenizedProjects) {
 	apply(from = "$rootDir/gradle/publishing.gradle.kts")
 
 	tasks.javadoc {
+		exclude("module-info.java")
```

Comment:
I wrote about excluding `module-info.java` from the `javadoc` task myself on [Twitter](https://twitter.com/sormuras/status/1114046969813532672), but I wonder: maybe doing this isn't necessary?

If you haven't set `--release` for the `javadoc` task (and it seems you haven't), it can handle `module-info.java` all right when building using JDK 9+.

Benefit: users can see the module info description in the JavaDocs, navigate over it, and generally feel it's a first-class citizen (even though the library still supports JDK 8).
Drawback: JavaDocs generated in such a way may perhaps look like they only support JDK 9+.